source/_integrations/generic.markdown

```diff
@@ -15,16 +15,11 @@ The `generic` camera platform allows you to integrate any IP camera or other URL
 
 Home Assistant will serve the images via its server, making it possible to view your IP cameras while outside of your network. The endpoint is `/api/camera_proxy/camera.[name]`.
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-To enable this camera in your installation, add the following to your `configuration.yaml` file:
+You must enter a URL in at least one of the fields **Still Image URL** or **Stream Source URL**, the others are optional.
 
-```yaml
-# Example configuration.yaml entry
-camera:
-  - platform: generic
-    still_image_url: http://194.218.96.92/jpg/image.jpg
-```
+[Templates](/topics/templating/) are allowed in the URL fields, which can be used to select different images or parameterize the URL depending on the status of sensors.  Template validity and network access are checked during the configuration steps.
 
 {% configuration %}
```

Comment:
Looks like this needs to be converted as well as it still links to how to find your `configuration.yaml`

CONFIGURATION VARIABLES
[Looking for your configuration file?](https://deploy-preview-14841--home-assistant-docs.netlify.app/docs/configuration/)

Reply:
Done